### PR TITLE
fix: install rocksdb from repo

### PR DESCRIPTION
**Pull request by InoMurko**
Slack raised issue and failed barebone build: https://app.circleci.com/jobs/github/omisego/elixir-omg/20131

## Overview

Install rocksdb via git

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- separate step in bin/setup

## Testing
barebuild

### CODE REVIEW

**Comment by pnowosie** [comment]
I recomend to suspend this - after running `bin/setup` I found my integration test fails localy...
Not sure if this is because `setup` I will investigate and update here

**Comment by InoMurko** [comment]
@pnowosie so the unit tests and integration tests now pass on debian 9. I don't think there's anything wrong with the installation script now

File: bin/setup [hunk 1]
```diff
@@ -62,6 +62,24 @@ install_linux_geth(){
...
+  echo "Installing Rocksdb v6.2.2"
+  mkdir tmp_rocksdb \
+  && cd tmp_rocksdb \
+  && git clone https://github.com/facebook/rocksdb.git \
+  && cd rocksdb  \
+  && git checkout v6.2.2 \
+  && make shared_lib \
+  && sudo mkdir -p /usr/local/rocksdb/lib \
+  && sudo mkdir -p /usr/local/rocksdb/include \
+  && sudo cp librocksdb.so* /usr/local/rocksdb/lib \
```
Comment (sirn) [inline bin/setup]: Maybe use `cp -P` so symlinks are not dereferenced

**Comment by pnowosie** [comment]
> @pnowosie so the unit tests and integration tests now pass on debian 9. I don't think there's anything wrong with the installation script now

Script is meant to be run on fresh envs it's not meant to be run on dev box we a lot of package are already installed, probably in diff versions.

**Comment by InoMurko** [comment]
Indeed @pnowosie. But that is something I'm unable to test.

**Pull request merged**